src/xrp-path-element.ts

```diff
@@ -0,0 +1,34 @@
+import { Payment } from './generated/web/org/xrpl/rpc/v1/transaction_pb'
+import XRPCurrency from './xrp-currency'
+
+type PathElement = Payment.PathElement
+
+/*
+ * A path in the XRP Ledger.
+ * - SeeAlso: https://xrpl.org/paths.html
+ */
+export default class XRPPathElement {
+  public account: string | null
+
+  public currency: XRPCurrency | null
+
+  public issuer: string | null
+
+  // TODO(amiecorso): is it ok to disable non-null assertion operator in these lines? (or perhaps for full file?)
+  // These fields are non-optional in the protobuf, so if for example hasAccount() is true, we would expect
+  // getAccount() to be non-null.
+  public constructor(pathElement: PathElement) {
+    this.account = pathElement.hasAccount()
+      ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        pathElement.getAccount()!.getAddress()
```

Comment:
I think you should use optional chaining rather than a non-null assertion. It's syntactic sugar:
```
pathElement.getAccount()?.getAddress()
```

Which says "if account is null, just stop and return null". In practice the above expands to:
```
const account = pathElement.getAccount()
if (!account) {
  return null
}
else account.getAddress()
```

You can do this indefinitely:
```
foo()?.bar()?.abc()?.xyz()
```